.travis.yml

```diff
@@ -1,5 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - "4"
-  - "6"
+    - "4"
```

Comment:
Why the increase in indentation? What about Node 9?

Reply:
Just copying what I've seen elsewhere